build.sh

```diff
@@ -219,6 +219,14 @@ then
             ext=*.dylib
 		fi	
 		cp  "${BuildOutputDir}/${__configuration}/Platform/${PublishDir}"/publish/${ext} "${__currentScriptDir}/src/python/nimbusml/internal/libs/"
+		# remove dataprep dlls as its not supported in python 2.7
+		rm -rf "${__currentScriptDir}/src/python/nimbusml/internal/libs/Microsoft.DPrep.*"
```

Comment:
The `-r` option is not needed here since there are no directories which are being removed.